builder-support/debian/recursor/debian-buster/control

```diff
@@ -8,8 +8,8 @@ Build-Depends: debhelper (>= 10),
                dh-systemd [linux-any],
                libboost-all-dev,
                libcap-dev,
-               libluajit-5.1-dev [!arm64],
-               liblua5.3-dev [arm64],
+               libluajit-5.1-dev [!arm64 !s390x],
```

Comment:
I prefer this form: https://github.com/PowerDNS/pdns/pull/9964/files#diff-3c0edaabae71305c965b62ac80ffa36de75c6919d543bd0d1dbee6db824731efR19-R20, can you copy that?

Reply:
@zeha tells me he's still experimenting with the right form.

Reply:
I'll merge this, @zeha has accepted that he'll get a small merge conflict.

Reply:
Just an FYI, you would need to include `ppc64el` in the example you posted. PowerPC has a JIT but it looks like the other supported architectures, s390x and arm64, do not.

Reply:
I expect #9964 to land soon with the exactly right predicates :)

Reply:
@james-crowley do you know if luajit _actually_works_ on ppc64el?

Reply:
@zeha Yup! I built the `auth` Dockerfile and ran the container. Had no issues. I think the confusion might be because its not "official". But a lot of distros thought it was of good enough quality and has included into their packages/repos. 

From reading the `luajit` repo, it looks like the dev is willing to support other architectures but does have the time/money to do so. So there are unofficial patches/branches for various architectures like `ppc64el` and `s390x`. Not sure why `s390x` did not get included into the distro's packages/repos, maybe due to lack of testing?